WinUIGallery/ItemPage.xaml

```diff
@@ -13,37 +13,25 @@
     x:Class="AppUIBasics.ItemPage"
```

Comment:
I think the spacing at the bottom (and maybe right) is a bit tight now:

![Screenshot of StandardUICommand page](https://user-images.githubusercontent.com/16122379/217065157-6874b5ac-3477-4816-b46d-c886d0236fa3.png)